data/mods/Magiclysm/items/herbs_and_flowers.json

```diff
@@ -0,0 +1,67 @@
+[
+
+  {
+    "type": "COMESTIBLE",
+    "id": "platonic_flower",
+    "name": { "str": "urflower", "str_pl": "urflowers" },
+    "weight": "60 g",
+    "color": "pink",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 400,
+    "description": "A fractal corolla winds inward in mathematical harmony, reflecting beauty in its ideal form.",
+    "price": 1000,
+    "material": "veggy",
+    "volume": "250 ml",
+    "fun": 8,
+    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ],
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "moonflower",
+    "copy-from": "platonic_flower",
+    "name": { "str": "moonflowers", "str_pl": "moonflowers" },
+    "color": "cyan",
+    "comestible_type": "FOOD",
+    "description": "A handful of moonflowers glitter like tiny stars and faintly smell of morning dew.  Properly harvested, they will never wilt or wither.",
+    "volume": "250 ml",
+    "use_action": "ROYAL_JELLY",
+    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ],
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "kämpuch",
+    "copy-from": "platonic_flower",
+    "name": { "str": "kämpuch", "str_pl": "kämpuch" },
+    "color": "purple",
+    "comestible_type": "FOOD",
+    "description": "A small rod of curled white petals.  Inhaling or ingesting the pollen within will lead to an agonizing if mercifully quick death.  Infamously used by romantic assassins, when an artistic death is deemed more important than practicality.",
+    "volume": "250 ml",
+    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ],
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "golden_reed_rhizome",
+    "copy-from": "platonic_flower",
+    "name": { "str": "golden reed rhizomes", "str_pl": "golden reed rhizomes" },
+    "color": "yellow",
+    "symbol": "%",
+    "calories": 800,
+    "comestible_type": "FOOD",
+    "description": "Starchy rhizomes from a golden reed.  They have a strong, spiced aroma.",
+    "volume": "250 ml",
+    "vitamins": [ [ "vitC", 5 ], [ "iron", 3 ], [ "calcium", 2 ] ],
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "moly_tuber",
+    "copy-from": "plaonic_flower",
+    "name": { "str": "moly tuber", "str_pl": "moly tubers" },
+    "color": "grey",
+    "symbol": "%",
+    "calories": 800,
+    "comestible_type": "FOOD",
+    "description": "The black root of a moly plant, extracted from the ground through great effort.  Will grant .",
```

Comment:
Will grant what O_o

Reply:
Dont worry much with reviewing this PR in its current state, its just a heavy WIP. Really I uploaded it only for self-motivation.

It will grant defenses tho if you wonder.